configurer/linux/flatcarcontainerlinux.go

```diff
@@ -0,0 +1,27 @@
+package linux
+
+import (
+	"github.com/k0sproject/rig"
+	"github.com/k0sproject/rig/os"
+	"github.com/k0sproject/rig/os/registry"
+)
+
+type FlatcatContainerLinux struct {
+	BaseLinux
+	os.Linux
+}
+
+func init() {
+	registry.RegisterOSModule(
+		func(os rig.OSVersion) bool {
+			return os.ID == "flatcarcontainerlinux"
```

Comment:
The OS ID of flatcar seems to be "flatcar".

```shell
sh-4.3# cat /etc/os-release
NAME="Flatcar Container Linux by Kinvolk"
ID=flatcar
ID_LIKE=coreos
VERSION=2955.0.0
VERSION_ID=2955.0.0
BUILD_ID=2021-08-04-0140
PRETTY_NAME="Flatcar Container Linux by Kinvolk 2955.0.0 (Oklo)"
ANSI_COLOR="38;5;75"
HOME_URL="https://flatcar-linux.org/"
BUG_REPORT_URL="https://issues.flatcar-linux.org"
FLATCAR_BOARD="amd64-usr"
```

Reply:
@kke Yes, detection works like a charm now. :wink: